install/cupy_builder/install_utils.py

```diff
@@ -21,3 +21,19 @@ def search_on_path(filenames):
             if os.path.exists(full):
                 return os.path.abspath(full)
     return None
+
+
+def generate_translation_unit(
+        func_name, type_name, code_name, source_path) -> str:
+    with open(source_path) as f:
```

Comment:
Can we make these files committed instead of generating them during installation, like done in Legate? I'm wondering dynamically generating files will make users difficult to track what code is actually running.

Reply:
Done in 8173f5c.